### PR TITLE
[MDEP-799] Add simple info json format for tree and cleanups

### PR DESCRIPTION
**Pull request by slawekjaranowski**
Only simple info for #391 - more detail documentation is still needed

### CODE REVIEW

File: src/main/java/org/apache/maven/plugins/dependency/tree/TreeMojo.java [hunk 1]
```diff
@@ -133,7 +107,8 @@ public class TreeMojo extends AbstractMojo {
 
     /**
      * If specified, this parameter will cause the dependency tree to be written using the specified format. Currently
-     * supported format are: <code>text</code> (default), <code>dot</code>, <code>graphml</code> and <code>tgf</code>.
+     * supported format are: <code>text</code> (default), <code>dot</code>, <code>graphml</code>, <code>tgf</code>
```
Comment (elharo) [inline src/main/java/org/apache/maven/plugins/dependency/tree/TreeMojo.java:110]: format --> formats

File: src/main/java/org/apache/maven/plugins/dependency/tree/TreeMojo.java [hunk 2]
```diff
@@ -86,30 +84,6 @@ public class TreeMojo extends AbstractMojo {
     @Parameter(property = "outputEncoding", defaultValue = "${project.reporting.outputEncoding}")
     private String outputEncoding;
 
-    /**
-     * Contains the full list of projects in the reactor.
-     */
-    @Parameter(defaultValue = "${reactorProjects}", readonly = true, required = true)
```
Comment (elharo) [inline src/main/java/org/apache/maven/plugins/dependency/tree/TreeMojo.java:92]: does this cause problems for existing projects that set these?

Comment (slawekjaranowski) [reply src/main/java/org/apache/maven/plugins/dependency/tree/TreeMojo.java:92]: no problem ... like unused variables are

**Review by elharo — APPROVED** [review]

**Comment by jira-importer** [comment]
Resolve #1310

**Pull request merged**